app/models/site.rb

```diff
@@ -62,12 +62,17 @@ def lon=(val)
     end
   end
 
+  # Returns true if the site geometry is null or has type 'ST_Point'.
+  def point?
+    geometry.nil? || geometry.geometry_type.type_name == 'Point'
+  end
+
```

Comment:
If this is meant to return true if geometry is a point, why is it true if geometry is nil? Is this primarily a way to ensure the geometry is not a line or complex shape?